components/automate-chef-io/content/docs/workflow_install.md

```diff
@@ -72,7 +72,7 @@ Then, on the Chef server:
 
 2. Creating a user in previous step displays a private key to stdout. Copy this content and and save it on the **Chef Automate** server as: `/hab/svc/automate-workflow-server/var/etc/delivery.pem`
 
-3. If this is a new chef-server, create an organization on the chef server with:
+3. If this is a new Chef Server, create an organization on the chef server with:
```

Comment:
This should be Chef Infra Server

Reply:
I'm leaving that for the docs team change, in order to avoid the inconsistency of the new term amid a sea of the old ones.